### PR TITLE
Support the TCP connection for OVSDB

### PR DESCRIPTION
**Pull request by hwchiu**
Support the TCP connection method for OVSDB and it should be applied to all ovs commands.

### CODE REVIEW

File: ovs/client.go [hunk 1]
```diff
@@ -317,6 +317,14 @@ func SetSSLParam(pkey string, cert string, cacert string) OptionFunc {
 	}
 }
 
+// SetTCPParam configures the OVSDB connection using a TCP format ip:port
+// for use with all ovs commands
+func SetTCPParam(ip string, port string) OptionFunc {
```
Comment (mdlayher) [inline ovs/client.go]: Combine these into a single parameter: `addr` that accepts the IP and string together.

File: ovs/client.go [hunk 2]
```diff
@@ -317,6 +317,14 @@ func SetSSLParam(pkey string, cert string, cacert string) OptionFunc {
 	}
 }
 
+// SetTCPParam configures the OVSDB connection using a TCP format ip:port
+// for use with all ovs commands
```
Comment (mdlayher) [inline ovs/client.go]: Specify "ovs-vsctl" instead of "ovs", and add a period at the end of the comment.

File: ovs/client.go [hunk 3]
```diff
@@ -317,6 +317,14 @@ func SetSSLParam(pkey string, cert string, cacert string) OptionFunc {
...
+func SetTCPParam(ip string, port string) OptionFunc {
+	return func(c *Client) {
+		c.flags= append(c.ofctlFlags, fmt.Sprintf("--db=tcp:%s:%s", ip, port))
```
Comment (mdlayher) [inline ovs/client.go]: This code isn't gofmt'd.

append's first argument should be `c.flags`.

**Comment by hwchiu** [comment]
Thanks your review and I have modified that according to your comments.

**Pull request merged**